helpdesk/settings.py

```diff
@@ -277,3 +277,19 @@
 # Attachment directories should be created with permission 755 (rwxr-xr-x)
 # Override it in your own Django settings.py
 HELPDESK_ATTACHMENT_DIR_PERMS = int(getattr(settings, 'HELPDESK_ATTACHMENT_DIR_PERMS', "755"), 8)
+
+def get_followup_webhook_urls():
+    urls = os.environ.get('HELPDESK_FOLLOWUP_WEBHOOK_URLS', None)
+    if urls:
+        return urls.split(',')
```

Comment:
Might be a good idea to cater for spaces before and after the comma here and in the new ticket function:
`return re_split(r"[\s],[\s]", urls)`